common/bin/oo-diagnostics

```diff
@@ -1226,6 +1228,28 @@ class OODiag
       do_warn "Using a self-signed certificate for the broker"
     end
 
+    # Retrieve the SSL cert from broker_host_env
+    env_response = `curl -k -s -v https://#{broker_host_env.chomp}/ -o /dev/null 2>&1`
+    if $? != 0
+      do_fail <<-CANNOTCONNECT
+        An error arose while trying to connect to the SSL reverse proxy.  curl returned with
+        #{env_response.strip.empty? ? "no output." : "the following output:\n" + env_response}
+
+        Please check whether the httpd service is running.
```

Comment:
In this error output, perhaps show that we attempted to connect to `#{broker_host_env}` and failed. Otherwise, the errors for `curl`ing localhost and `#{broker_host_env}` are the same.